appinfo/info.xml

```diff
@@ -5,7 +5,7 @@
     <summary>Integration of GitLab software development management service</summary>
     <description><![CDATA[GitLab integration provides a dashboard widget displaying your most important notifications
     and a unified search provider for repositories, issues and merge requests.]]></description>
-    <version>1.0.18</version>
+    <version>1.0.19</version>
```

Comment:
Personally, I'd bump the minor version for adding support for a new nextcloud version, but it's up to you :)